admin/load.php

```diff
@@ -266,10 +280,11 @@ function( $a, $b ) {
  * @since 1.0.0
  *
  * @param string $module_file Absolute path to the main module file.
+ * @param string $module_dir  The module directory ({focus}/{module-slug}).
  * @return array|bool Associative array of parsed module data, or false on failure. Fields for every module include
  *                    'name', 'description', 'focus', and 'experimental'.
  */
-function perflab_get_module_data( $module_file ) {
+function perflab_get_module_data( $module_file, $module_dir ) {
 	$default_headers = array(
```

Comment:
This still needs to be adjusted to remove the `focus` field since it's no longer in the module header.

Reply:
removed in [`86f7553` (#58)](https://github.com/WordPress/performance/pull/58/commits/86f755346278b3473f727a836af52a99a354576e)